Docs: add a note about workspace or account-level provider usage (#3074)
databricks/terraform-provider-databricks: docs/resources/metastore_assignment.md

---
# databricks_metastore_assignment (Resource)

> -> **Note** This resource could be used with account or workspace-level provider.

Reviewer: we only officially support this for account-level
```suggestion
-> **Note** This resource could be only used with account-level provider!
```
